src/rosecommon/src/croseisc.cpp

```diff
@@ -90,7 +90,7 @@ bool CRoseISC::onReceived(uint16_t& packet_size_, uint8_t* buffer_) {
   return rtnVal;
 }
 
-bool CRoseISC::onSend(uint8_t* _buffer) {
+bool CRoseISC::onSend([[maybe_unused]] uint16_t socket_id_, [[maybe_unused]] uint8_t* _buffer) {
   // TODO: Encrypt the isc buffer.
   (void)_buffer;
```

Comment:
you don't need the `(void)_buffer;` here if you mark it as `[[maybe_unused]]`

Reply:
I know, I just left it the way it was.